src/main.rs

```diff
@@ -507,7 +534,7 @@ mod tests {
                     .trim()
                     .parse::<u32>()
                     .unwrap(),
-                8
+                10
```

Comment:
We probably want to define that one as a constant, and reuse it across different tests. The change when adding a new MSI based device would also look more readable.